<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>start</class>
 <widget class="QDialog" name="start">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1252</width>
    <height>867</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1241</width>
     <height>851</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>ajouter</string>
    </attribute>
    <widget class="QPushButton" name="pb_ajouter">
     <property name="geometry">
      <rect>
       <x>1070</x>
       <y>480</y>
       <width>141</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>ajouter</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_id">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_des">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>60</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Destination</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>90</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>mode de paiement :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_offre">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>220</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_client">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>260</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateEdit">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>300</y>
       <width>110</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>230</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>id_offre</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>270</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>id_client</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>300</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>date</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_billet">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>480</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>billet</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame_billet">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>510</y>
       <width>351</width>
       <height>331</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>num billet</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_num_billet">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>20</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QDateTimeEdit" name="dateTimeEdit_vente">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>50</y>
        <width>194</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QDateTimeEdit" name="dateTimeEdit_aller">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>80</y>
        <width>194</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QDateTimeEdit" name="dateTimeEdit_retour">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>110</y>
        <width>194</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit_delai">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>140</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_prix">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>190</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_etat">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>230</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_type">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>270</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>date de vente</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>date d'aller</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>date de retour</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>delai d'annulation</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>190</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>prix</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>230</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>etat</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>270</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>type</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_guide">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>380</y>
       <width>221</width>
       <height>61</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLineEdit" name="lineEdit_guide">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>20</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>id_guide</string>
      </property>
     </widget>
    </widget>
    <widget class="QCheckBox" name="checkBox_guide">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>340</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>guide</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_pic">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>15</y>
       <width>1221</width>
       <height>801</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_billet">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>460</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_hotel">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>650</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_3">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>670</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>hotel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_guide">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>320</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QFrame" name="frame_hotel">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>710</y>
       <width>321</width>
       <height>61</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>id_hotel</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_hotel">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QCheckBox" name="checkBox_4">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>790</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>transport</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_supprimer">
     <property name="geometry">
      <rect>
       <x>1070</x>
       <y>570</y>
       <width>141</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>supprimer</string>
     </property>
    </widget>
    <widget class="QPushButton" name="mise_a_jour">
     <property name="geometry">
      <rect>
       <x>1070</x>
       <y>660</y>
       <width>141</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>update</string>
     </property>
    </widget>
    <widget class="QPushButton" name="afficher">
     <property name="geometry">
      <rect>
       <x>1070</x>
       <y>740</y>
       <width>141</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>afficher</string>
     </property>
    </widget>
    <widget class="QTabWidget" name="tabWidget_2">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>10</y>
       <width>931</width>
       <height>441</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Tableau des reservations</string>
      </attribute>
      <widget class="QTableView" name="tabreservation_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>911</width>
         <height>401</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>Tableau des billets</string>
      </attribute>
      <widget class="QTableView" name="tabbillet">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>921</width>
         <height>401</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QRadioButton" name="espece">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>95</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>espece</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="cheque">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>95</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>cheque</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="facilite">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>120</y>
       <width>95</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>facilité</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="carte_banquire">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>150</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>carte banquire</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>90</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_transport">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>760</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>label_pic</zorder>
    <zorder>pb_ajouter</zorder>
    <zorder>lineEdit_id</zorder>
    <zorder>lineEdit_des</zorder>
    <zorder>label</zorder>
    <zorder>label_2</zorder>
    <zorder>label_3</zorder>
    <zorder>lineEdit_offre</zorder>
    <zorder>lineEdit_client</zorder>
    <zorder>dateEdit</zorder>
    <zorder>label_7</zorder>
    <zorder>label_8</zorder>
    <zorder>label_10</zorder>
    <zorder>checkBox_billet</zorder>
    <zorder>frame_billet</zorder>
    <zorder>frame_guide</zorder>
    <zorder>checkBox_guide</zorder>
    <zorder>label_hotel</zorder>
    <zorder>checkBox_3</zorder>
    <zorder>label_billet</zorder>
    <zorder>label_guide</zorder>
    <zorder>frame_hotel</zorder>
    <zorder>checkBox_4</zorder>
    <zorder>pb_supprimer</zorder>
    <zorder>mise_a_jour</zorder>
    <zorder>afficher</zorder>
    <zorder>tabWidget_2</zorder>
    <zorder>espece</zorder>
    <zorder>cheque</zorder>
    <zorder>facilite</zorder>
    <zorder>carte_banquire</zorder>
    <zorder>lineEdit_3</zorder>
    <zorder>label_transport</zorder>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>afficher</string>
    </attribute>
    <widget class="QTableView" name="tabreservation">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>671</width>
       <height>311</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>supprimer</string>
    </attribute>
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
    <widget class="QPushButton" name="A">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>330</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>supprimer</string>
     </property>
    </widget>
    <widget class="QPushButton" name="B">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>330</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>mettre a jour</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
